payments-ui-core/src/main/java/com/stripe/android/ui/core/PaymentsTheme.kt

```diff
@@ -104,6 +120,20 @@ fun PaymentsThemeConfig.toComposeColors(): PaymentsComposeColors {
     )
 }
 
+@Composable
+@ReadOnlyComposable
+@RestrictTo(RestrictTo.Scope.LIBRARY_GROUP)
+fun PaymentsThemeConfig.toComposeShapes(): PaymentsComposeShapes {
+    val shapes = PaymentsThemeConfig.ShapeConfig
```

Comment:
I think `PaymentsThemeConfig.ShapeConfig` looks kind of strange. I would expect to see something like `PaymentsTheme.ShapeConfig` or something along the lines of this. What do you think?

Reply:
I think I made this less weird. Take a look